### PR TITLE
Improve Red Night floating cam style

### PR DESCRIPTION
**Pull request by Poeschl**
Improving the floating cam window a little bit ;)

If you don't mind add a `hacktoberfest-accepted` label to the PR bevor merging.
Or add the `hacktoberfest` label to your repositories to show that your projects are participating on this year [Hacktoberfest](https://hacktoberfest.digitalocean.com/).

### CODE REVIEW

**Comment by LazeMSS** [comment]
Done...

**Comment by LazeMSS** [comment]
@Poeschl are you participating? And what repos will you be working on?

**Comment by Poeschl** [comment]
@LazeMSS Yes, i am. 😉 Right now I can't say for sure. I guess it will be something around home assistant and maybe my own addons.
I'm also host of a hacktoberfest event at my company gathering people to work together on a little riddle we are developing right now.

**Comment by LazeMSS** [comment]
I added a couple of comments in a review - I don't like making changes to other peoples themes ;)

**Comment by Poeschl** [comment]
Also for the red-night? Didn't see them so far. Might be worth a look. 😁

**Comment by Poeschl** [comment]
Orrrr, how is the color variable thing going?

**Comment by LazeMSS** [comment]
60% done :)

**Comment by LazeMSS** [comment]
> Also for the red-night? Didn't see them so far. Might be worth a look. 😁

I just wrote it as review for you: 
https://github.com/LazeMSS/OctoPrint-UICustomizerThemes/pull/30/files/4aaa1ed318dde1bd51aeba83b25678079bd0cf5e

**Comment by Poeschl** [comment]
Don't see anything 😅

File: css/red-night.css [hunk 1]
```diff
@@ -842,14 +842,23 @@ hr {
...
-#UICWebCamFull {
-    background-color: var(--grey1);
+#UICWebCamFull.FloatCam {
+    background-color: var(--background);
     transition: box-shadow .2s ease-in-out;
     box-shadow: 0 0 8px 4px var(--quiteWhite-transparent);
+    padding: 0;
+}
+
+#UICWebCamFull.FloatCam .label {
```
Comment (LazeMSS) [inline css/red-night.css:852]: @Poeschl see this?

Remove the floatcam

**Pull request merged**